### DISCUSSION TITLE
host.docker.internal alternative

### DISCUSSION THREAD

**Discussion by electriquo** [rancher-sandbox/rancher-desktop#1483] [Q&A] [answered]
I am migrating from Docker Desktop to Rancher Desktop (RD). Right now, I am using RD version 1.0.1 with dockerd/moby.
Is there any alternative to [`host.docker.internal`](https://docs.docker.com/desktop/mac/networking/#use-cases-and-workarounds)

> #### I want to connect from a container to a service on the host
> 
> The host has a changing IP address (or none if you have no network access). We recommend that you connect to the special DNS name `host.docker.internal` which resolves to the internal IP address used by the host. This is for development purpose and will not work in a production environment outside of Docker Desktop for Mac.

```shell
$ docker run --rm curlimages/curl -vs http://host.docker.internal:8000
* Could not resolve host: host.docker.internal
* Closing connection 0
```

Is it missing from Rancher Desktop?
What is the alternative?

**Comment by jandubois** [comment, marked as answer]
We define the name inside the VM, but it currently does not resolve from inside containers. https://github.com/lima-vm/lima/issues/622 will add this for macOS and Linux, and https://github.com/rancher-sandbox/rancher-desktop/pull/1302 will provide this on Windows.

**Reply by electriquo** [reply]
In the meanwhile, is there any alternative?

**Reply by dchambers** [reply]
`192.168.5.2` worked for me, as per:

- https://github.com/lima-vm/lima/blob/master/docs/network.md

**Reply by jandubois** [reply]
> `192.168.5.2` worked for me, as per:

yes, this will work for Linux and macOS, but not for Windows.
